lib/uksched/sched.c

```diff
@@ -286,3 +287,9 @@ void uk_sched_thread_exit(void)
 	uk_sched_thread_remove(thread->sched, thread);
 	UK_CRASH("Failed to stop the thread\n");
 }
+
+UK_SYSCALL_R_DEFINE(int, sched_yield)
```

Comment:
Besides the `newlib` conflict previously mentioned, there also seems to be a conflict when compiling with `pthread-embedded`. To be more precise, it conflicts with the declaration located in the [extracted source code](https://github.com/unikraft/lib-pthread-embedded/blob/c59d7d632c9fc50f8ebe4e24139ff01265f14f9c/Makefile.uk#L189)